src/modules/commands/discord/urbandictionary.ts

```diff
@@ -0,0 +1,82 @@
+import { Command } from '@commands/CommandHandler'
+import got from 'got'
+import { isRecord } from '@app/validators/object'
+import camelcaseKeys from 'camelcase-keys'
+import discord from 'discord.js'
+
+type Definition = {
+  definition: string
+  permalink: string
+  thumbsUp: number
+  soundUrls: string[]
+  author: string
+  word: string
+  defid: number
+  currentVote: string
+  writtenOn: string
+  example: string
+  thumbsDown: number
+}
+
+export default new Command<discord.Message>({
+  keyword: 'urbandictionary',
+  description: 'get the modern definition of a word or expression',
+  help: 'Usage: `{prefix}urbandictionary` - get the modern definition of a word or expression',
+  options: {
+    lastArgNumber: 1,
+  },
+  callback: async ({ args, reply, env }) => {
+    if (args.length === 0) return reply(env.command.getHelp(env.handler))
+
+    const word = args[0]
+
+    const term = word
```

Comment:
Why do you have `word` and `term` when they are the same? (I prefer term)

Reply:
hahahahha